contents/docs/feature-flags/bootstrapping-and-local-evaluation.mdx

```diff
@@ -58,10 +64,18 @@ Calling this function forces PostHog to hit the endpoint for the updated informa
 
 ## Server-side local evaluation
 
-If you're using our server-side libraries, you can use local evaluation to improve performance instead of making additional API requests. This requires:
+If you're using our server-side libraries, you can use local evaluation to improve performance instead of making additional API requests. 
+
+We consider it best practice to use local evaluation flags where possible because this method enables you to resolve flags faster. This is especially true if you’re using multiple flags at once — instead of sending a `decide` request per flag and waiting for each call to resolve, you simply serve the correct flags all at once based on the definition data. 
+
+Local evaluation calls are also more efficient. Instead of calling the API for each user, you can re-use the definitions for multiple users. 
+
+> For billing purposes we count a local evaluation as being equivalent to 10 `decide`requests. Users generally retrieve the payloads of hundreds or thousands of users at once, so this ensures local evaluation is priced fairly but normally remains the most cost-effective option by far. 
```

Comment:
The local evaluation request downloads what the flag says it needs to match with. So for example, if your feature flag was: "Enable for all users in USA", what the local evaluation request downloads is: "Flag 1 = 'Enable for all users in USA'".

In contrast, with decide request, you give it a user ID, and it returns "Flag 1 = 'true or false', Flag 2 = 'true or false', etc. etc. "

```suggestion
> For billing purposes we count a local evaluation API request as being equivalent to 10 `decide` requests. One local evaluation request can compute feature flags for hundreds or thousands of users, so this ensures local evaluation is priced fairly but normally remains the most cost-effective option by far. 
```